gramps/plugins/importer/importxml.py

```diff
@@ -866,7 +866,7 @@ class object of a primary object.
                     "event": self.db.has_event_handle,
                     "place": self.db.has_place_handle,
                     "source": self.db.has_source_handle,
-                    "citation": self.db.get_raw_citation_data,
+                    "citation": self.db.has_citation_handle,
```

Comment:
@Nick-Hall this one might be worth worth pushing to 6.0 as well. Let me know and I can make a separate PR

I think it is more of a performance problem, unnecessarily get the citation `DataDict`, just to check if the handle is a valid citation

Reply:
Yes.  Something like this is obviously a mistake and just a single line change.  I think that we can accept it as a bug even if it isn't actually causing a problem.

Reply:
OK. I'll split out later.